ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -159,6 +159,205 @@ objects:
         mkdir -p /tmp/output
         cd /tmp/output
 
+        function restore-cluster-state() {
+          echo "Placing file /etc/rollback-test with contents A"
+          cat > /tmp/machineconfig.yaml <<'EOF'
+          apiVersion: machineconfiguration.openshift.io/v1
+          kind: MachineConfig
+          metadata:
+            labels:
+              machineconfiguration.openshift.io/role: master
+            name: 99-rollback-test
+          spec:
+            config:
+              ignition:
+                config: {}
+                security:
+                  tls: {}
+                timeouts: {}
+                version: 2.2.0
+              networkd: {}
+              passwd: {}
+              storage:
+                files:
+                - contents:
+                    source: data:,A
+                    verification: {}
+                  filesystem: root
+                  mode: 420
+                  path: /etc/rollback-test
+              systemd: {}
+        EOF
+          oc create -f /tmp/machineconfig.yaml
+
+          function wait_for_machineconfigpool_to_apply() {
+
+            for i in $(seq 0 10); do oc wait machineconfigpool/master --for=condition=Updating --timeout=5m && break; done
+            for i in $(seq 0 10); do oc wait machineconfigpool/master --for=condition=Updated --timeout=5m && break; sleep 30; done
```

Comment:
"Try up to 10 times" is not very robust.  For example, a network hiccup could burn us through several attempts in a very short period of time.  And is the `Updating` check racy?  What happens if we update very quickly and the state is `Updated` before the `Updating` wait comes through?  As long as this test isn't blocking, I'm fine cutting these corners, but I think we want to tighten this up before we make this blocking.

Reply:
>What happens if we update very quickly and the state is Updated before the Updating wait comes through?

Update requires a master reboot, so I'd not worry about that

Reply:
>For example, a network hiccup could burn us through several attempts in a very short period of time

In other places I tried to put more sleeps, but yes, a better generic solution is required

Reply:
Implemented in 94c0348d